substrate/frame/child-bounties/src/lib.rs

```diff
@@ -184,16 +192,22 @@ pub mod pallet {
 		Canceled { index: BountyIndex, child_index: BountyIndex },
 	}
 
-	/// Number of total child bounties.
+	/// DEPRECATED: Replaced with `ParentTotalChildBounties` storage item keeping dedicated counts
+	/// for each parent bounty. Number of total child bounties. Will be removed in May 2025.
 	#[pallet::storage]
 	pub type ChildBountyCount<T: Config> = StorageValue<_, BountyIndex, ValueQuery>;
```

Comment:
Why do we need to keep this around? I dont see a use for it. Or is it generally to minimize breaking changes?

Reply:
The second